### PR TITLE
Updated Vertiq2306 with new pulsing firmware. Added additional files …

### PR DESCRIPTION
**Pull request by Vertiq-Ben**
…for backwards compatibility

- Added custom_iq_module because the old pulsing_module.py depended on it.

### CODE REVIEW

File: iqmotion/iq_devices/custom_iq_module.py
Comment (iq-fred) [inline iqmotion/iq_devices/custom_iq_module.py:1]: Note: This was on Bitbucket, but did not make it to Github yet. Adding it to keep them in sync and allow us to use the existing, backwards compatible Pulsing Module

File: iqmotion/iq_devices/module_files/pulsing.json
Comment (iq-fred) [inline iqmotion/iq_devices/module_files/pulsing.json:1]: All of these pulsing additions already exist on Bitbucket, just moving them up to Github to keep them in sync and release it to the public

File: iqmotion/iq_devices/vertiq2306_module.py
Comment (iq-fred) [inline iqmotion/iq_devices/vertiq2306_module.py:1]: Same changes as on this Bitbucket PR: https://bitbucket.org/iqcontrol/python_api/pull-requests/48

**Review by iq-fred — APPROVED** [review]

File: iqmotion/iq_devices/pulsing_module.py
Comment (iq-fred) [inline iqmotion/iq_devices/pulsing_module.py:1]: Same as on Bitbucket, keeping this around for "backwards compatibility" with users that were previously sent preliminary versions of pulsing API stuff

**Comment by Vertiq-Ben** [comment]
Corresponding BitBucket PR: https://bitbucket.org/iqcontrol/python_api/pull-requests/48

**Pull request merged**